src/client/search/TermItem.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+
+const TermItem = ({ onDeleteClick, onToggleParamTypeClick, query, source, paramTypes, id }) => (
+  <div id={`paramTypes${id}`} className="ui multiple compact dropdown labeled icon violet button"
```

Comment:
Hmmmmmm, instead of appending the `id` to the id field, would a data-id={id} perhaps make more sense? Easier to search for and select programatically too